src/generated/method-types.ts

```diff
@@ -1,5 +1,5 @@
 import type { EndpointInterface, RequestInterface } from "@octokit/types";
-import type { RestEndpointMethodTypes } from "./parameters-and-response-types.js";
+import type { RestEndpointMethodTypes } from "./parameters-and-response-types";
```

Comment:
@wolfy1339 It's not ideal that this is auto-removed. Perhaps we should rework our automation a little bit so those changes are ignored.

Reply:
that would need to be changed here:
https://github.com/octokit/plugin-rest-endpoint-methods.js/blob/56bab3e531ebe1c5477efdc1f58fddb0b666cbb4/scripts/update-endpoints/types.mjs#L90

Reply:
Thank you! I've added that change on to this PR.

Reply:
Woops, didn't catch that one. Thanks for fixing it